server/events/models/models_test.go

```diff
@@ -612,3 +612,31 @@ func TestPullStatus_StatusCount(t *testing.T) {
 	Equals(t, 1, ps.StatusCount(models.ErroredPolicyCheckStatus))
 	Equals(t, 1, ps.StatusCount(models.PassedPolicyCheckStatus))
 }
+
+func TestPolicySetDataList_CompressDecompress(t *testing.T) {
+	cases := []struct {
```

Comment:
Add a test case that triggers the panic compressing or decompressing the conftest output.

Reply:
I can probably get a test to trigger a panic on decompressing. I'm not exactly sure how to get it to trigger for compressing.